src/ROUTES.ts

```diff
@@ -89,6 +89,7 @@ const ROUTES = {
     SETTINGS_PRONOUNS: 'settings/profile/pronouns',
     SETTINGS_PREFERENCES: 'settings/preferences',
     SETTINGS_SUBSCRIPTION: 'settings/subscription',
+    SETTINGS_SUBSCRIPTIONS_SIZE: 'settings/subscription/subscription-size',
```

Comment:
```suggestion
    SETTINGS_SUBSCRIPTION_SIZE: 'settings/subscription/subscription-size',
```

@MrMuzyk Minor typo I noticed while working on my design doc